pinot-query-planner/src/main/java/org/apache/pinot/query/planner/nodes/SerDeUtils.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.query.planner.nodes;
+
+import org.apache.pinot.common.proto.Plan;
+
+
+public final class SerDeUtils {
```

Comment:
We should ideally merge this with `ProtoSerializationUtils` code to keep all related `SerDe` code together

Reply:
yes and no. the ProtoSerializationUtils are mainly for StageNodes, this is more for the entire payload sent across the wire. thus this would have context regarding the runtime/server/cluster setup, where the StageNode only deal with planner. but yes I will make it a better naming to differentiate the intent of both classes.